_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -1536,6 +1572,7 @@ mfxStatus VAAPIEncoder::CreateAuxilliaryDevice(
         const auto sliceCapabilities = AV(VAConfigAttribEncSliceStructure);
         const auto sliceStructure = sliceCapabilities & ~VA_ENC_SLICE_STRUCTURE_MAX_SLICE_SIZE;
         m_caps.ddi_caps.SliceStructure = ConvertSliceStructureVAAPIToMFX(sliceStructure);
+        m_caps.ddi_caps.SliceLevelRateCtrl = sliceCapabilities & VA_ENC_SLICE_STRUCTURE_MAX_SLICE_SIZE ? 1 : 0;
```

Comment:
Please avoid hardcode (like 1/0). Give name them

Reply:
SliceLevelRateCtrl is binary flag. So using 1/0 is reasonable for this case. Even if we try to give meaningful name, it would be "enabled/disabled", "true/false", "supported/unsupported" and so on, the same meaning to 1/0

Reply:
Ahh, it's just bool:
https://github.com/Intel-Media-SDK/MediaSDK/blob/bcaea9ec4aca9ac8c08a524d40403c609c143a1a/_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h#L241

I thought it's enum. Then okay.